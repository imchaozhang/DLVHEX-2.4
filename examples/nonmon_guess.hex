% this program should do a nondeterministic guess across the
% nonmonotonic external atom testSetMinus
seta(u). seta(v). seta(w).

setc(X) :- &testSetMinus[seta,setd](X), seta(X).
setd(Y) :- &testSetMinus[seta,setc](Y), seta(Y).
