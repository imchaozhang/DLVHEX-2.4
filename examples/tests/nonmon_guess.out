{setd(w), setd(v), setd(u), seta(w), seta(v), seta(u)}
{setd(w), setd(v), setc(u), seta(w), seta(v), seta(u)}
{setd(w), setd(u), setc(v), seta(w), seta(v), seta(u)}
{setd(w), setc(v), setc(u), seta(w), seta(v), seta(u)}
{setd(v), setd(u), setc(w), seta(w), seta(v), seta(u)}
{setd(v), setc(w), setc(u), seta(w), seta(v), seta(u)}
{setd(u), setc(w), setc(v), seta(w), seta(v), seta(u)}
{setc(w), setc(v), setc(u), seta(w), seta(v), seta(u)}
